src/Microsoft.OData.Client/ODataProtocolVersion.cs

```diff
@@ -10,6 +10,9 @@ namespace Microsoft.OData.Client
     public enum ODataProtocolVersion
     {
         /// <summary>Version 4</summary>
-        V4
+        V4,
+
+        /// <summary>Version 4.01</summary>
+        V401

```

Comment:
Maybe V4_01 be better

Reply:
CA complained about underscores in enums.

---
In reply to: [150044112](https://github.com/OData/odata.net/pull/989#discussion_r150044112) [](ancestors = 150044112)